src/employees/components/UpdateEmployeeForm.vue

```diff
@@ -0,0 +1,224 @@
+        <v-card-title>Contact info</v-card-title>
+        <v-card-text>
+          <v-row>
+            <v-col>
+              <v-text-field
+                v-model="employeeCopy.email"
+                :rules="emailRules"
+                label="Email"
+                required
+                validate-on-blur
+                outlined
+              ></v-text-field>
+            </v-col>
+
+            <v-col>
+              <v-text-field
+                v-model="employeeCopy.phoneNumber"
+                label="Phone number"
+                type="phone"
+                :rules="phoneNumberRules"
+                required
+                outlined
+                validate-on-blur
+              ></v-text-field>
+            </v-col>
+          </v-row>
+
+          <v-row>
+            <v-col>
+              <v-text-field
+                v-model="employeeCopy.address"
+                label="Address"
+                required
+                outlined
+              ></v-text-field>
+            </v-col>
+
+            <v-col>
+              <v-menu
+                v-model="calendarOpen"
+                :close-on-content-click="false"
+                transition="scale-transition"
+                offset-y
+                max-width="290px"
+                min-width="auto"
+              >
+                <template v-slot:activator="{ on, attrs }">
+                  <v-text-field
+                    v-model="employeeCopy.birthDate"
+                    label="Birth date"
+                    outlined
+                    persistent-hint
+                    append-icon="mdi-calendar"
+                    readonly
+                    v-bind="attrs"
+                    v-on="on"
+                  ></v-text-field>
+                </template>
+                <v-date-picker
+                  v-model="employeeCopy.birthDate"
+                  no-title
+                  @input="calendarOpen = false"
+                ></v-date-picker>
+              </v-menu>
+            </v-col>
+          </v-row>
+
+          <v-row>
+            <v-col cols="12" md="1">
+              <v-switch
+              :true-value="ENABLED_STATUS"
+              :false-value="DISABLED_STATUS"
+              v-model="employeeCopy.status"
+              :label="employeeCopy.status | status"
+              ></v-switch>
+            </v-col>
+          </v-row> 
+        </v-card-text>
+      </v-card>
+
+      <v-row justify="space-between">
+        <v-col cols="12" md="3">
+          <v-btn outlined color="#72418b" @click="cancelUpdateEmployee()" class="mr-5"
+            >CANCEL</v-btn
+          >
+
+          <v-btn color="#72418b" dark type="submit" @click="validate()" class="mr-5"
+            >SAVE</v-btn
+          >
+        </v-col>
+
+        <v-col cols="12" md="2">
+          <v-btn outlined color="#F44336" @click="confirmDeleteEmployee(employeeCopy.id)"
+            >DELETE</v-btn
+          >
+        </v-col>
+      </v-row>
```

Comment:
We should align the delete button to the right 
![image](https://user-images.githubusercontent.com/9523581/126955411-8da4b77c-e4ca-40c6-aa4a-e771ff380d9c.png)


simple yet best solution

```suggestion
      <div class="d-flex justify-space-between">
        <div>
          <v-btn outlined color="#72418b" @click="cancelUpdateEmployee()" class="mr-5"
            >CANCEL</v-btn
          >

          <v-btn color="#72418b" dark type="submit" @click="validate()" class="mr-5"
            >SAVE</v-btn
          >
        </div>

        <v-btn outlined color="#F44336" @click="confirmDeleteEmployee(employeeCopy.id)"
          >DELETE</v-btn
        >
      </div>
```

Reply:
I believe the component `CreateEmployeeForm` can be reused for updates, just rename it to `EmployeeForm` and update small things to binding data to UI (in edit mode), therefore we don't need the `UpdateEmployeeForm.vue`

in the CreateEmployee page, we will have
`<EmployeeForm @submit="onCreateEmployee" />`

in the UpdateEmployee page we will have
`<EmployeeForm :employee="employee" @submit="onUpdateEmployee" @delete-employee="onDeleteEmployee"/>`